contrib/init/sysvinit-redhat/balena-engine.sysconfig

```diff
@@ -1,4 +1,4 @@
-# /etc/sysconfig/balena
+# /etc/sysconfig/balena-engine
```

Comment:
Note that both the file and the service were renamed (`balena.sysconfig` → `balena-engine.sysconfig`).